integration/yarn/src/main/java/alluxio/yarn/YarnUtils.java

```diff
@@ -147,5 +149,17 @@ public static String buildCommand(YarnContainerType containerType, Map<String, S
     return commandBuilder.toString();
   }
 
+  /**
+   * Get yarn resource manager ip address.
+   *
+   * @param config yarn configuration
+   * @return the ip address of yarn resource manager
+   */
+  public static InetSocketAddress getRMAddress(Configuration config) {
+    return config.getSocketAddr(YarnConfiguration.RM_ADDRESS,
+            YarnConfiguration.DEFAULT_RM_ADDRESS,
```

Comment:
NIT: the indent doesn't look correct to me (should be 4 space on line breaks), please checkout http://www.alluxio.org/documentation/v1.0.0/en/Contributing-to-Alluxio.html#coding-style